### PR TITLE
wip: script for verifying restored receipts

### PR DESCRIPTION
**Pull request by Longarithm**

### CODE REVIEW

**Comment by bowenwang1996** [comment]
Why is this a separate PR from #4248 ?

File: utils/restored-receipts-verifier/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        apply_transactions_for_not_included_chunk(&runtime, shard_id, &chunk_extra, &block);
+        receipts_missing_after_apply = apply_result.receipt_result.values().cloned().into_iter().flatten().collect();
+        receipts_missing.extend(receipts_missing_after_apply.into_iter());
+        println!("{} applied", height);
+    }
+
+    let bytes = receipts_missing.try_to_vec().unwrap();
+    let str_path = String::from("./utils/restored-receipts-verifier/receipts_missing.dat");
+    let rxs_path = Path::new(&str_path);
+    std::fs::write(rxs_path, bytes);
```
Comment (matklad) [inline utils/restored-receipts-verifier/src/main.rs:57]:
```suggestion
    std::fs::write("./utils/restored-receipts-verifier/receipts_missing.dat", bytes);
```

Comment (Longarithm) [reply utils/restored-receipts-verifier/src/main.rs:57]: Thank you!

**Comment by Longarithm** [comment]
@bowenwang1996 I use this branch and PR for testing a script on 1.18.3 branch, because it doesn't work on master with current mainner dumps.

**Comment by bowenwang1996** [comment]
Is this PR still needed? If not, please close it

**Pull request closed without merging**